Microsoft/WebTemplateStudio
# Azure - simplify forms

After talking with our users and sponsors, we decided to remove the resource group field from both Cosmos DB and App Service forms, and remove the location field from the Cosmos DB form.

**Before**:
![image](https://user-images.githubusercontent.com/20333350/63378164-7768e180-c346-11e9-8a00-a64ba67074ae.png)
![image](https://user-images.githubusercontent.com/20333350/63378174-818ae000-c346-11e9-95ef-b56646b717cb.png)
___
**After**:
![image](https://user-images.githubusercontent.com/20333350/63377993-1214f080-c346-11e9-8057-8af8ce97728e.png)
![image](https://user-images.githubusercontent.com/20333350/63378017-1f31df80-c346-11e9-83e5-02c9e1ac4d61.png)

___
**To test:**
- Create a project, add Cosmos DB and App Service
- After deployment is done, go to the portal and find your resource group
- Make sure that the Cosmos DB and App Service is in the resource group
- Both services should be in Central US
